_includes/manuals/3.12/3.2.2_installer_options.md

```diff
@@ -341,7 +350,7 @@ kafo-export-params -c /etc/foreman-installer/scenarios.d/foreman.yaml -f html
       </tr>
       <tr>
         <td style='white-space:nowrap'>--apache-mod-status-requires</td>
-        <td>A Variant type that can be: - String with: - '' or 'unmanaged' - Host auth control done elsewhere - 'ip &lt;List of IPs>&gt;' - Allowed IPs/ranges - 'host &lt;List of names&gt;' - Allowed names/domains - 'all [granted|denied]' - Array of strings with ip or host as above - Hash with following keys: - 'requires' - Value => Array as above - 'enforce' - Value => String 'Any', 'All' or 'None' This encloses "Require" directives in "<Require(Any|All|None)>" block Optional - If unspecified, "Require" directives follow current flow</td>
+        <td>A Variant type that can be: - String with: - '' or 'unmanaged' - Host auth control done elsewhere - 'ip <List of IPs>' - Allowed IPs/ranges - 'host <List of names>' - Allowed names/domains - 'all [granted|denied]' - Array of strings with ip or host as above - Hash with following keys: - 'requires' - Value => Array as above - 'enforce' - Value => String 'Any', 'All' or 'None' This encloses "Require" directives in "<Require(Any|All|None)>" block Optional - If unspecified, "Require" directives follow current flow</td>
```

Comment:
Oh yes, this was the issue with escaping which the script didn't do. Though it surprises me we didn't need to escape the `<` and `>` in `Require`. HTML can probably deal with it because of the weird tags in between. Please keep this as it was.